include/tim/vx/tensor.h

```diff
@@ -106,7 +106,7 @@ struct TensorSpec {
 
   TensorSpec& SetDataType(DataType datatype);
 
-  TensorSpec& SetShape(ShapeType& shape);
+  TensorSpec& SetShape(const ShapeType& shape);
```

Comment:
Maybe you also need modify the associated traced API.

Reply:
The args is derived by DEF_TRACED_API automatically.
https://github.com/VeriSilicon/TIM-VX/blob/265e74ff1668d21214c67c360c0bef91c44ca7fb/include/tim/experimental/trace/tvx/tensor.h#L152-L152